commons-test/src/main/java/org/infinispan/commons/test/CommonsTestingUtil.java

```diff
@@ -47,6 +47,17 @@ public static String tmpDirectory() {
       return System.getProperty("infinispan.test.tmpdir", System.getProperty("java.io.tmpdir"));
    }
 
+   /**
+    * Creates a path to a unique (per test) temporary directory. The directory is created in the platform's temp
+    * directory (set by {@code java.io.tmpdir}).
+    *
+    * @param test test that requires this directory.
+    * @return an absolute path
+    */
+   public static String javaTmpDirectory(Class<?> test) {
```

Comment:
Why is this necessary? Can't we just use `tmpDirectory(Class<?> test)`?

Reply:
This is for the JPA store. 
The path has to be set in `persistence.xml` but the `tmpDirectory()` method uses `infinispan.test.tmpdir`. If `infinispan.test.tmpdir` is not set, it falls back to `java.io.tmpdir`. 
If I use `infinispan.test.tmpdir` and it was not set, the DB will point to the incorrect location.